gateways/clients/responses/hathor_wallet_service/hathor_wallet_service_token_response.py

```diff
@@ -0,0 +1,161 @@
+    """Response from hathor wallet service when requesting a token
+
+    :param id: id of token
+    :type id: str
+    :param name: token name
+    :type name: str
+    :param symbol: token symbol
+    :type symbol: str
+    :param timestamp: when token was created
+    :type timestamp: int
+    :param total_supply: token total supply
+    :type total_supply: int
+    :param can_mint: if token have mint authority or not
+    :type can_mint: bool
+    :param can_melt: if token have melt authority or not
+    :type can_melt: bool
+    """
+    id: str
+    name: str
+    symbol: str
+    timestamp: int
+    total_supply: int
+    can_mint: bool
+    can_melt: bool
+
+    @classmethod
+    def from_dict(cls, dikt: dict) -> 'HathorWalletServiceTokenResponse':
+        return from_dict(data_class=cls, data=dikt)
+
+    def to_token(self) -> Token:
+        """Convert wallet service token response to a domain token
+
+        :raises Exception: If the token is in wrong format
+        :return: the Token generated
+        :rtype: :py:class:`domain.tx.token.Token`
+        """
+        return Token(
+            id=self.id,
+            name=self.name,
+            symbol=self.symbol,
+            total_supply=self.total_supply,
+            can_melt=self.can_melt,
+            can_mint=self.can_mint
+        )
+
+
+@dataclass
+class HathorWalletServiceTokenTransaction:
+    """Transaction Item from a transaction list requested to hathor wallet service
+
+    :param id: id hash of the transaction
+    :type id: str
+
+    :param timestamp: timestamp of transaction creation
+    :type timestamp: int
+    """
+    id: str
+    timestamp: int
+
+
+@dataclass
+class HathorWalletServiceTokenTransactionsReponse:
+    """Response from hathor wallet service when requesting a list of token transactions
+
+    :param total: total number of transactions of a given token
+    :type total: int
+    :param transactions: list of transactions
+    :type transactions: List[:py:class:`domain.tx.TokenTransactionListItem`]
+    """
+    total: int
+    transactions: List[HathorWalletServiceTokenTransaction]
+
+    @classmethod
+    def from_dict(cls, dikt: dict) -> 'HathorWalletServiceTokenTransactionsReponse':
+        return from_dict(data_class=cls, data=dikt)
+
+    def to_token_transaction_list(self) -> TokenTransactionList:
+        """Convert wallet service response to a domain token transaction list
+
+        :raises Exception: If the token transacton list is in wrong format
+        :return: the new TokenTransactionList generated
+        :rtype: :py:class:`domain.tx.token.TokenTransactionList`
+        """
+        try:
+            return TokenTransactionList(
+                total=self.total,
+                transactions=[TokenTransactionListItem(
+                    id=transaction.id,
+                    timestamp=transaction.timestamp
+                ) for transaction in self.transactions]
+            )
+        except Exception:
```

Comment:
It feel dangerous to catch `Exception`. Why is that? It would capture ANY exception generated inside it (even if your code is broken).